x-pack/plugin/vectors/src/test/java/org/elasticsearch/xpack/vectors/mapper/DenseVectorFieldMapperTests.java

```diff
@@ -89,6 +107,53 @@ public void testDefaults() throws Exception {
         );
     }
 
+    public void testAddDocumentsToIndexBefore_V_7_4_0() throws Exception {
+        Version indexVersion = Version.V_7_3_0;
```

Comment:
looks like we also need to update the version in tests

Reply:
Thanks again, I've been having a sleepy day.